<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="0" failed="1" skipped="2">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-01-21T02:32:05 CST" name="Default Suite" finished-at="2023-01-21T02:32:05 CST" duration-ms="51">
    <groups>
    </groups>
    <test started-at="2023-01-21T02:32:05 CST" name="TestNGExample" finished-at="2023-01-21T02:32:05 CST" duration-ms="51">
      <class name="com.automation.test.CartPageTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.automation.test.CartPageTest@a74868d]" started-at="2023-01-21T02:32:05 CST" name="setUp" finished-at="2023-01-21T02:32:05 CST" duration-ms="7" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="verifyUserCanRemoveItemFromCart()[pri:0, instance:com.automation.test.CartPageTest@a74868d]" started-at="2023-01-21T02:32:05 CST" name="verifyUserCanRemoveItemFromCart" finished-at="2023-01-21T02:32:05 CST" duration-ms="1" status="FAIL">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[Test Failed!]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: Test Failed!
at com.automation.test.CartPageTest.verifyUserCanRemoveItemFromCart(CartPageTest.java:9)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyUserCanRemoveItemFromCart -->
        <test-method is-config="true" signature="cleanUp()[pri:0, instance:com.automation.test.CartPageTest@a74868d]" started-at="2023-01-21T02:32:05 CST" name="cleanUp" finished-at="2023-01-21T02:32:05 CST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanUp -->
        <test-method signature="verifyUserCanAddItemIntoCart()[pri:0, instance:com.automation.test.CartPageTest@a74868d]" started-at="2023-01-21T02:32:05 CST" name="verifyUserCanAddItemIntoCart" finished-at="2023-01-21T02:32:05 CST" depends-on-methods="com.automation.test.CartPageTest.verifyUserCanRemoveItemFromCart" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method CartPageTest.verifyUserCanAddItemIntoCart()[pri:0, instance:com.automation.test.CartPageTest@a74868d] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method CartPageTest.verifyUserCanAddItemIntoCart()[pri:0, instance:com.automation.test.CartPageTest@a74868d] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:101)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyUserCanAddItemIntoCart -->
        <test-method signature="verifyUserCanChangeQuantityOfItemOnCart()[pri:0, instance:com.automation.test.CartPageTest@a74868d]" started-at="2023-01-21T02:32:05 CST" name="verifyUserCanChangeQuantityOfItemOnCart" finished-at="2023-01-21T02:32:05 CST" depends-on-methods="com.automation.test.CartPageTest.verifyUserCanAddItemIntoCart" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method CartPageTest.verifyUserCanChangeQuantityOfItemOnCart()[pri:0, instance:com.automation.test.CartPageTest@a74868d] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method CartPageTest.verifyUserCanChangeQuantityOfItemOnCart()[pri:0, instance:com.automation.test.CartPageTest@a74868d] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:101)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyUserCanChangeQuantityOfItemOnCart -->
      </class> <!-- com.automation.test.CartPageTest -->
    </test> <!-- TestNGExample -->
  </suite> <!-- Default Suite -->
</testng-results>
